src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Tooltip from 'box-ui-elements/es/components/tooltip/Tooltip';
+import { withTargetedClickThrough } from 'box-ui-elements/es/features/targeting/hocs';
+import { ControlsLayerContext } from '../controls-layer';
+import { ExperiencesContext } from '../experiences';
+import { TargetingApi } from '../../../types';
+import './AnnotationsTargetedTooltip.scss';
+
+export type Props = {
+    children: React.ReactElement;
```

Comment:
It looks like we're using React.ReactNode in some places and React.ReactElement in others. Should we use the `React.PropsWithChildren` helper?

Reply:
I would like to. However when I use `React.PropsWithChildren` or `children: React.ReactNode` & `return children`. I get the following type error:
```
Type 'ReactNode' is not assignable to type 'Element | null'.
 Type 'undefined' is not assignable to type 'Element | null'.
```
It does work when I use `React.PropsWithChildren` & `return <>children</>`
It also is working when I use `children: React.ReactElement` & `return children`